### PR TITLE
fix: [CDS-79969]: Fix account invite redirection when FF is enabled

### PR DESCRIPTION
**Pull request by MondalSayantan**
### Summary

When a user is added to an account and the person clicks on "accept invite", originally they are redirected to `/home/get-started`. In new nav, this is redirected to `/admin/get-started` which does not exist. Created a route which redirects to Admin overview page.

#### Screenshots

https://github.com/harness/harness-core-ui/assets/62999423/7777537f-cb3b-45a5-9379-d181acc1cb07

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by MondalSayantan** [comment]
retrigger jest

File: src/modules/45-projects-orgs/AdminRouteDestinations.tsx [hunk 1]
```diff
@@ -51,7 +51,8 @@ const AdminRouteDestinations = (): React.ReactElement => {
         path={[
           routes.toMode({ ...projectPathProps, mode: NAV_MODE.ADMIN }),
           routes.toMode({ ...orgPathProps, mode: NAV_MODE.ADMIN }),
-          routes.toMode({ mode: NAV_MODE.ADMIN, ...accountPathProps })
+          routes.toMode({ mode: NAV_MODE.ADMIN, ...accountPathProps }),
+          routes.toLandingDashboard({ ...accountPathProps })
```
Comment (avinashmadhwani02) [inline src/modules/45-projects-orgs/AdminRouteDestinations.tsx:55]: <RouteWithContext path={routes.toLandingDashboard({ ...accountPathProps })}>
        // add redirect here
  </RouteWithContext>

File: src/modules/45-projects-orgs/AdminRouteDestinations.tsx [hunk 2]
```diff
@@ -56,6 +56,9 @@ const AdminRouteDestinations = (): React.ReactElement => {
       >
         <AdminRedirect />
       </RouteWithContext>
+      <RouteWithContext exact path={routes.toLandingDashboard({ ...accountPathProps })}>
+        <Redirect to={routes.toOverview()} />
+      </RouteWithContext>
       <RouteWithContext path={routes.toOverview({ ...accountPathProps, ...modePathProps })}>
         <LandingDashboardPage />
       </RouteWithContext>
```
Comment (Sagar-Rajput7) [inline src/modules/45-projects-orgs/AdminRouteDestinations.tsx:64]:
```suggestion
      <RouteWithContext path={[routes.toLandingDashboard({ ...accountPathProps, ...modePathProps }), routes.toOverview({ ...accountPathProps, ...modePathProps })]}>
        <LandingDashboardPage />
      </RouteWithContext>
```

This should also do the trick

Comment (MondalSayantan) [reply src/modules/45-projects-orgs/AdminRouteDestinations.tsx:64]: This won't work as we need redirection from `get-started` to `overview` and not just rendering of page

**Comment by MondalSayantan** [comment]
retrigger cypress-cv

**Review by avinashmadhwani02 — APPROVED** [review]

**Review by Sagar-Rajput7 — APPROVED** [review]

**Pull request merged**